src/Domain/Uuid/validateUuid.ts

```diff
@@ -0,0 +1,24 @@
+import { ValidatedValue } from '../Validation/Validation'
+import { Uuid } from './Uuid'
+import { validate } from 'uuid'
+
+/**
+ * Checks if a value of type `any` (e.g. from a request) is a valid UUID.
+ */
+// eslint-disable-next-line @typescript-eslint/no-explicit-any, @typescript-eslint/explicit-module-boundary-types
+export function validateUuid(uuid: any): ValidatedValue<Uuid> {
```

Comment:
As mentioned before - I don't think we need this - wdyt?

Reply:
Just delegates to `validate` from the `uuid` library with one additional check in case `uuid` is not defined or has the wrong type. This consistently complements the `CreateValetTokenValidation` functions. At this point I'm pretty sure it would be difficult to generate a request that passes the validation but does not conform to the static types. And this gives an even stronger guarantee on the `uuid`s -- that they are not just strings but well-formatted strings. So no code that deals with the validated values has to worry about these errors.

That said all this validation may be overdone for our purposes, since, come to think of it, the requests with these `dto`s will come from (I'm assuming now, but didn't previously -- hence this validation) a trusted source which is `auth`.

Anyway since it's already in place maybe leave it all in and start removing only if and as we will be introducing changes to `CreateValetKeyDto`?

Reply:
Sounds good 👍